src/explore-education-statistics-admin/src/services/preReleaseUserService.ts

```diff
@@ -15,7 +15,7 @@ const preReleaseUserService = {
   },
   removeUser(releaseId: string, email: string): Promise<void> {
     return client.delete(`/release/${releaseId}/prerelease-users`, {
-      data: {
+      params: {
```

Comment:
Don't think query params is a great way of doing this as it leaves emails in your browsing history. I'm surprised that `data` wasn't working!

Reply:
OK will take a look, ta